packages/react/src/exampleTypes/TypeMap.js

```diff
@@ -9,6 +9,7 @@ import TagsText from './TagsText.js'
 import Exists from './Exists.js'
 import Bool from './Bool.js'
 import Text from './Text.js'
+import StepSlider from './StepSlider'
```

Comment:
Does this cause an error?
```suggestion
import StepSlider from './StepSlider.tsx'
```

Reply:
Correct! And to solve that I would need to wrestle a bit with the tooling to understand and solve.
![Screenshot 2024-08-19 at 10 56 26](https://github.com/user-attachments/assets/2cbd0ec5-fe3c-4a06-80d4-c019b7683652)

Given my constraints, I would rather skip this fight for now.

Reply:
+1 to this. Sounds good. I'll add a follow-up